### PR TITLE
Typo alert rebase

### PR DESCRIPTION
**Pull request by kel29**

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -101,6 +101,11 @@ <h2 id='game-over-header' class="hidden">Game Over!</h2>
               </div>
             </div>
 
+            <div class="row justify-content-center">
+              <div id="typo-alert" class="hidden">
```
Comment (renburnett) [inline frontend/index.html]: maybe this could be changed to an alert modal thingy in the future?

File: frontend/js/wordInteractions.js [hunk 2]
```diff
@@ -98,12 +93,12 @@ function incrementTimerOnScreen () {
...
 
-  const secs = Math.round((currentTime - startTime) / 1000) % 60;
-  secs < 10 ? seconds.textContent = 0 + secs : seconds.textContent = secs;
-  const mins = Math.round((currentTime - startTime) / 60000) % 60;
-  mins < 10 ? minutes.textContent = 0 + mins : minutes.textContent = mins;
-  const hrs = Math.round((currentTime - startTime) / 360000) % 24;
-  hrs < 10 ? hours.textContent = 0 + hrs : hours.textContent = hrs;
+  const secs = Math.floor((currentTime - startTime) / 1000) % 60;
+  secs < 10 ? seconds.textContent = '0' + secs : seconds.textContent = secs;
+  const mins = Math.floor((currentTime - startTime) / 60000) % 60;
```
Comment (renburnett) [inline frontend/js/wordInteractions.js]: good catch with the `math.round` and `math.floor` switch

File: frontend/index.html [hunk 3]
```diff
@@ -101,6 +101,11 @@ <h2 id='game-over-header' class="hidden">Game Over!</h2>
...
+              </div>
+            </div>
+
```
Comment (justinamaple) [inline frontend/index.html]: Might not have followed this in the index myself, but anything within a row should be within a col. This is due to the negative padding that rows have, which is balanced by the positive padding within cols.

File: frontend/js/wordInteractions.js [hunk 4]
```diff
@@ -136,6 +132,30 @@ function playerTypesWord () {
...
+  h4.textContent = `TYPO: "${typedSubmission}" is not a valid entry!`;
+
+  let lives = 'lives';
+  if (game.typos === 2) { lives = 'life'; }
+  p.textContent = `You have ${3 - game.typos} ${lives} remaining.`;
+
+  thisTypoDiv.append(h4, p);
+  allTyposDiv.appendChild(thisTypoDiv);
+  setTimeout(removeTypoAlert, 2500);
+}
```
Comment (justinamaple) [inline frontend/js/wordInteractions.js]: Look into using the MDB alert class instead

File: frontend/index.html (hunk 1 above)
Comment (kel29) [reply frontend/index.html]: ooh have you been using that for the leaderboard? New to me, would love to learn

File: frontend/index.html (hunk 3 above)
Comment (kel29) [reply frontend/index.html]: I'll play with this as I look into styling for the alert, good tip

File: frontend/js/wordInteractions.js (hunk 4 above)
Comment (kel29) [reply frontend/js/wordInteractions.js]: will do!

**Pull request merged**